app/components/Header/Header.css

```diff
@@ -113,61 +113,223 @@
   align-items: center;
   flex: 2;
   white-space: nowrap;
-  margin-right: 5px;
+  margin-right: 0.3em;
 
   @media (--mobile-device) {
     display: none;
   }
 }
 
-.navigation a {
-  font-size: 1.35rem;
-  padding: 10px 1rem;
-  height: 100%;
-  color: var(--color-dark-gray-5);
+.navbarEl {
+  margin: auto;
+}
+
+.navbarList {
+  display: flex;
+  justify-content: center;
+  list-style: none;
+  margin: 0;
+}
+
+.navbarItemEl {
+  position: relative;
+  width: max-content;
+}
+
+.navbarItemTitle {
+  height: 2rem; /* TODO: optimize */
+  background: transparent;
+  border: 0;
+  font-family: -apple-system, system-ui, 'Segoe UI', 'Roboto', 'Helvetica Neue',
```

Comment:
The audacity to add a new font 😱  
![image](https://user-images.githubusercontent.com/23152018/147395059-f15c21a0-646c-4439-8dad-fdb5a975c50f.png)

... and not just any font
> Segoe UI — Segoe UI ("User Interface") is a member of the Segoe family used in Microsoft products for user interface text

@LudvigHz gonna be mad

Reply:
Yeah, like the other font, this is really, really unnecessary and will just slow the page down for a very slight (worse) font for three words, and you have the extra css of setting the right font in the dropdown containers as well.

Please, just no...

Reply:
Yeah, like the other font, this is really, really unnecessary and will just slow the page down for a very slight (worse) font for three words, and you have the extra css of setting the right font in the dropdown containers as well.

Please, just no...